web/src/ProviderTable.js

```diff
@@ -193,6 +193,28 @@ class ProviderTable extends React.Component {
       //     )
       //   }
       // },
+      {
+        title: i18next.t("application:rule"),
```

Comment:
```suggestion
        title: i18next.t("application:Rule"),
```

Reply:
@leo220yuyaodog I found that `application:rule` is also used in `SignupTable.js`, do I need to modify it together?

Reply:
> @leo220yuyaodog I found that `application:rule` is also used in `SignupTable.js`, do I need to modify it together?

Yes, modify together.